pages/reference/OS/updates/update-process.md

```diff
@@ -3,32 +3,108 @@ title: Update process details
 
+### Automatic rollbacks
 Rollbacks is a framework designed to automatically roll back the OS update in case something goes wrong.
 
 There are two rollback mechanisms in the OS, covering different update failure modes. One based on health checks called the `rollback-health`, and the recognizing if the new system is unbootable for some reason, the `rollback-altboot`. Learn more about [rollback mechanisms][rollback-mechanism].
 
+
+## Advanced HUP
+As mentioned above, Advanced HUP is usually required for certain very specific upgrade paths. The underlying reasons for incompatibility with Standard HUP can be, but not limited to:
+
+1. Partition layout changes from one OS version to another. For example:
+    
+    -  The partition layout for NVIDIA JetPack 4-based {{ $names.os.lower }} differs from JetPack 5-based {{ $names.os.lower }}. If a device is running already running JetPack-4 {{ $names.os.lower }}, you can use Advanced HUP to update to a JetPack 5-based {{ $names.os.lower }}
+    - {{ $names.company.upper }} releases a new version of an OS for a certain device type whereby the root partition has been increased to accommodate more kernel module and drivers
+    -  {{ $names.company.upper }} releases a new version of an OS for a certain device type to include new functionality such as Secure Boot
+2. Changing OS architecture, for example: 
+    - Switching from a the 32-bit OS variant running on Raspberry Pi 3 to a 64-bit variant of {{ $names.os.lower }}
+
+__Note:__ If you've used Advanced HUP, it does not mean necessarily updates won't be compatible with Standard HUP. For instance, if a device running 32-bit  {{ $names.os.lower }} on a Raspberry Pi 3 has been updated to a 64-bit variant of {{ $names.os.lower }}, Standard HUP can be used for future updates to the 64-bit OS. In this case, Advanced HUP is only used once to migrate the device from one OS architecture to another.
+
+### Caveats
+When using the Advanced HUP, it’s important to be aware of several limitations compared to the Standard HUP mechanism.
+
+#### Data loss
+Data loss may occur, as the process can involve overwriting partitions and filesystems. Please make sure to backup all data on the device before running any update. Also, after a successful update, all application containers will be downloaded again.
+
+#### No automatic rollbacks
+Advanced HUP does not provide a built-in rollback mechanism. If the new OS version proves unstable or introduces unexpected issues, there is no automated way to revert to a previous state. 
+
+#### No delta updates
+The OS image is downloaded directly via the ap. The registry and {{ $names.engine.lower }} is not used during the download process.
+
+### How it works
+Similar to Standard HUP, an updater script gets transferred to the device to handle the update process.
```

Comment:
The docs are better kept atomic, referring to a previous section might not be required here. Since we are fully describing the process. 

If you accepted the previous suggestion, then you can make the same improvement here.